template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -52,6 +53,11 @@
 yourself; then run `run-cbmc-proofs --no-standalone`; add any
 additional jobs that you want to execute with `litani add-job`; and
 finally run `litani run-build`.
+
+The litani reports will be located under the `output` directory. The
```

Comment:
Would you mind changing this sentence to "The litani reports will be located under the output directory (the cbmc-viewer reports will remain in the reports directory)." The problem is that the word "report" is used all over, and because "make report" has traditionally meant make the cbmc-viewer report, the word report is becoming ambiguous.

Reply:
Changed text below. I think using "dashboard" for litani and "report" for cbmc-viewer is probably helpful, I'll start using that terminology from now on.
> The litani dashboard will be written under the `output` directory; the
cbmc-viewer reports remain in the `$PROOF_DIR/report` directory. The
HTML dashboard from the latest Litani run will always be symlinked to
`output/latest/html/index.html`, so you can keep that page open in
your browser and reload the page whenever you re-run this script.